index.html

```diff
@@ -14,183 +14,7 @@
 
     <title>Frontend Mentor | Fylo landing page with two column layout</title>
     <link rel="stylesheet" href="style.css" />
-    <style>
-      .attribution {
-        font-size: 11px;
-        text-align: center;
-      }
-      .attribution a {
-        color: hsl(228, 45%, 44%);
-      }
-    </style>
   </head>
   <body>
-    <header>
-      <div class="icon">
-        <img src="./images/logo.svg" alt="Logo" />
-      </div>
-      <nav class="navbar">
-        <div class="hamburger-icon">
-          <img src="./images/menu-hamburger.svg" alt="" onclick="openMenu()" />
-        </div>
-        <div class="menu" id="menu">
-          <ul>
-            <li><a href="#">Features</a></li>
-            <li><a href="#">Team</a></li>
-            <li><a href="#">Sign In</a></li>
-          </ul>
-        </div>
-      </nav>
-    </header>
-
-    <main>
-      <section class="hero">
-        <div class="hero-img">
-          <img src="/images/illustration-1.svg" alt="illustration-1" />
-        </div>
-        <div class="hero-content">
-          <h1>All your files in one secure location, accessible anywhere.</h1>
-          <p>
-            Fylo stores your most important files in one secure location. Access
-            them wherever you need, share and collaborate with friends, family,
-            and co-workers.
-          </p>
-          <div class="hero-content-input-btn">
-            <input
-              type="email"
-              name="email"
-              id="email"
-              placeholder="Enter your email"
-            />
-            <button class="hero-content-btn">Get Started</button>
-          </div>
-        </div>
-      </section>
-
-      <section class="section-two">
```

Comment:
It's always tempting to name elements with numbers to differentiate them, but it makes it harder for others (and yourself in the future) to know which section this is. A good alternative is to use a descriptive name based on what is inside that section - in this case it could be something like "section-how-it-works"